tests/Functional/ContentCallForProposalsRenderTest.php

```diff
@@ -0,0 +1,379 @@
+
+/**
+ * Tests that "Call for proposals" content type renders correctly.
+ */
+class ContentCallForProposalsRenderTest extends ContentRenderTestBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  public static $modules = [
+    'config',
+    'datetime_testing',
+    'block',
+    'system',
+    'path',
+    'oe_theme_helper',
+    'oe_theme_content_call_proposals',
+  ];
+
+  /**
+   * {@inheritdoc}
+   */
+  protected function setUp(): void {
+    parent::setUp();
+
+    // Give anonymous users permission to view entities.
+    Role::load(RoleInterface::ANONYMOUS_ID)
+      ->grantPermission('view published skos concept entities')
+      ->grantPermission('view published oe_contact')
+      ->save();
+  }
+
+  /**
+   * Tests full page of Call for proposals.
+   */
+  public function testProposalRendering(): void {
+    // Freeze the time at a specific point.
+    $static_time = new DrupalDateTime('2020-02-17 14:00:00', DateTimeItemInterface::STORAGE_TIMEZONE);
+    $publication_date = (clone $static_time)->modify('- 5 days');
+
+    /** @var \Drupal\Component\Datetime\TimeInterface $datetime */
+    $time = \Drupal::time();
+    $time->freezeTime();
+    $time->setTime($static_time->getTimestamp());
+
+    // Create a Call for proposal node with required fields only.
+    /** @var \Drupal\node\Entity\Node $node */
+    $node = $this->getStorage('node')->create([
+      'type' => 'oe_call_proposals',
+      'title' => 'Test Call for proposals node',
+      'oe_publication_date' => [
+        'value' => $publication_date->format(DateTimeItemInterface::DATE_STORAGE_FORMAT),
+      ],
+      'oe_call_proposals_model' => 'permanent',
+      'oe_content_content_owner' => 'http://publications.europa.eu/resource/authority/corporate-body/COMMU',
+      'uid' => 0,
+      'status' => 1,
+    ]);
+    $node->save();
+    $this->drupalGet($node->toUrl());
+
+    // Assert page header - metadata.
+    $page_header = $this->assertSession()->elementExists('css', '.ecl-page-header-core');
+    $assert = new PatternPageHeaderAssert();
+    $header_expected_values = [
+      'title' => 'Test Call for proposals node',
+      'meta' => 'Call for proposals',
+    ];
+    $assert->assertPattern($header_expected_values, $page_header->getOuterHtml());
+
+    // Assert navigation part.
+    $navigation = $this->assertSession()->elementExists('css', 'nav.ecl-inpage-navigation');
+    $inpage_nav_assert = new InPageNavigationAssert();
+    $inpage_nav_expected_values = [
+      'title' => 'Page contents',
+      'list' => [
+        ['label' => 'Details', 'href' => '#details'],
+      ],
+    ];
+    $inpage_nav_assert->assertPattern($inpage_nav_expected_values, $navigation->getOuterHtml());
+
+    // Assert content part.
+    $wrapper = $this->assertSession()->elementExists('css', '.ecl-row.ecl-u-mt-l');
+    $content = $this->assertSession()->elementExists('css', '.ecl-col-lg-9', $wrapper);
+    $this->assertSession()->elementsCount('css', '.ecl-col-lg-9', 1);
```

Comment:
Do we need this assertion here? And if there is only one, why do we need the $wrapper to get the $content?